### PR TITLE
nfstestserver: change export path to use persistent disk

### PR DESCRIPTION
**Pull request by T0FFF**
Add property nfstestserver.export_root_path with default value /var/vcap/store

### CODE REVIEW

**Comment by gberche-orange** [comment]
Thanks @T0FFF 

Suggesting PR test refinement to add a bit more context:

----

The [nfs-test-server](https://github.com/cloudfoundry/nfs-volume-release#deploying-the-test-nfs-server-optional) is convenient to run an experimental NFS server, but currently does not store data on the [provisionned bosh persistent disk](https://github.com/cloudfoundry/nfs-volume-release/blob/master/templates/nfs-test-server-manifest-aws.yml#L29). As a result, stored data is ephemeral and lost among bosh vm recreation.

This PR adds a property ``nfstestserver.export_root_path`` with default value ``/var/vcap/store`` making the NFS share persistent by default. Setting this property to "" restores the current default behavior of storing persistent data on the root disk at path ``/export/``.

---------

It might make sense to submit the PR to ``cloudfoundry:master`` from a branch name more explicit than ``orange-cloudfoundry:master`` (e.g. ``orange-cloudfoundry:nfstestserver-persistent-export``)

/CC @nabbar any other feedback ?

File: jobs/nfstestserver/spec [hunk 1]
```diff
@@ -11,4 +11,6 @@ properties:
  nfstestserver.export_cidr:
     description: "cidr range to use for nfs exports"
     default: "10.0.0.0/8"
-
+ nfstestserver.export_root_path:
+    description: "NFS root path"
+    default: "/var/vcap/store"
```
Comment (gberche-orange) [inline jobs/nfstestserver/spec]: The convention used in other releases is to add the job name to the path, e.g. ``/var/vcap/store/mysql`` in https://github.com/cloudfoundry/cf-mysql-release/blob/develop/jobs/mysql/templates/my.cnf.erb#L67

The default value might then benefit from being ``/var/vcap/store/nfstestserver``

**Review by gberche-orange — CHANGES_REQUESTED** [review]
Please consider improving default path

File: jobs/nfstestserver/spec (hunk 1 above)
Comment (T0FFF) [reply jobs/nfstestserver/spec]: It's a very good point. Thanks @gberche-orange

**Pull request merged**